.github/workflows/firebase-pull-request.yml

```diff
@@ -0,0 +1,21 @@
+# This file was auto-generated by the Firebase CLI
```

Comment:
Delete this entire file, the functionality will be taken over by the single firebase workflow file